docs/samples/uri/README.md

```diff
@@ -0,0 +1,233 @@
+# Predict on a `InferenceService` with a saved model from a URI
+This allows you to specify a model object via the URI (Uniform Resource Identifier) of the model object exposed via an `http` or `https` endpoint. 
+
+This `storageUri` option supports single file models, like `sklearn` which is specified by a [joblib](https://joblib.readthedocs.io/en/latest/) file, or artifacts (e.g. `tar` or `zip`) which contain all the necessary dependencies for other model types (e.g. `tensorflow` or `pytorch`). Here, we'll show examples from both of the above.
+
+## Setup
+1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/#install-kfserving).
+2. Your cluster's Istio Ingress gateway must be network accessible.
+3. Your cluster's Istio Egress gateway must [allow http / https traffic](https://knative.dev/docs/serving/outbound-network-access/)
+
+## Sklearn
+### Train and freeze the model
+Here, we'll train a simple iris model. Please note that `kfserving` requires `sklearn==0.20.3`. 
+
+```python
+from sklearn import svm
+from sklearn import datasets
+import joblib
+
+def train(X, y):
+    clf = svm.SVC(gamma='auto')
+    clf.fit(X, y)
+    return clf
+
+def freeze(clf, path='../frozen'):
+    joblib.dump(clf, f'{path}/model.joblib')
+    return True
+
+if __name__ == '__main__':
+    iris = datasets.load_iris()
+    X, y = iris.data, iris.target
+    clf = train(X, y)
+    freeze(clf)
+```
+Now, you'll need to take that frozen model object and put it somewhere on the web to expose it. For instance, pushing the `model.joblib` file to some repo on GitHub.
+
+### Specify and create the `InferenceService`
+```yaml
+apiVersion: serving.kubeflow.org/v1alpha2
+kind: InferenceService
+metadata:
+  name: sklearn-from-uri
+spec:
+  default:
+    predictor:
+      sklearn:
+        storageUri: https://github.com/tduffy000/kfserving-uri-examples/blob/master/sklearn/frozen/model.joblib?raw=true
+
+```
+
+Apply the CRD,
+```bash
+kubectl apply -f sklearn_uri.yaml
+```
+Expected Output
+```
+$ inferenceservice.serving.kubeflow.org/sklearn-from-uri created
+```
+### Run a prediction
+Now, if everything went according to plan you should be able to hit the endpoint exposing the model we just uploaded.
```

Comment:
the ingress access instruction has been updated recently, can you help take a look at [tensorflow example](https://github.com/kubeflow/kfserving/tree/master/docs/samples/tensorflow#run-a-prediction) and change accordingly ?

Reply:
yes, will follow the posted example and update it